src/__tests__/update-logs.test.ts

```
@@ -0,0 +1,392 @@
+
+		// Override LOG_FILE for tests to use our test file
+		originalLogFile = process.env.LOG_FILE;
+		process.env.LOG_FILE = TEST_LOG_FILE;
+	});
+
+	afterEach(async () => {
+		// Restore original LOG_FILE
+		if (originalLogFile) {
+			process.env.LOG_FILE = originalLogFile;
+		} else {
+			delete process.env.LOG_FILE;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)